.readthedocs.yaml

```
@@ -2,22 +2,16 @@
 # See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
 
 version: 2
-
 build:
-  os: ubuntu-22.04
+  os: ubuntu-20.04
```

```suggestion
  os: ubuntu-22.04
```

no need to downgrade here